modules/custom/social_demo/src/DemoContentParser.php

```diff
@@ -18,7 +18,7 @@ public function getPath($file, $module, $profile) {
     if ($profile != '' && strpos($profile, DIRECTORY_SEPARATOR) === FALSE) {
       $profile .= DIRECTORY_SEPARATOR;
     }
-    return drupal_get_path('module', $module) . DIRECTORY_SEPARATOR . $profile . $file;
+    return \Drupal::service('extension.list.module')->getPath($module) . DIRECTORY_SEPARATOR . $profile . $file;
```

Comment:
@navneet0693 the DemoContentParser is a service, so why not use dependency injection?

Reply:
Check okay.